blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -174,8 +174,13 @@
     // 3 days - this matches the TTL for our metrics_full table, we don't accept anything older than the TTL.
     BEFORE_CURRENT_COLLECTIONTIME_MS("259200000"),
     // 10 minutes
-    AFTER_CURRENT_COLLECTIONTIME_MS("600000");
+    AFTER_CURRENT_COLLECTIONTIME_MS("600000"),
 
+    // Cross-Origin Resource Sharing
+    CORS_ALLOWED_ORIGINS(""),
```

Comment:
@VinnyQ so I am ok with keeping the config here. The default of the configuration property can be "". But in the code that actually sends back the headers, we should check this config. If the config is "", then don't send back any CORS headers. Sending ```Access-Control-Allow-Origin: ``` with empty value is illegal.